AlphaWallet/Tokens/Types/TokenObject.swift

```diff
@@ -111,3 +118,51 @@ class TokenObject: Object {
         return .init(chainID: chainId)
     }
 }
+
+class TokenInfoObject: Object {
+    @objc dynamic var uid: String = ""
+    @objc dynamic var coinGeckoId: String?
```

Comment:
Are these just 2 values that are used to store inside `TokenObject` in the database? Why do we need them now (and not before)?

Reply:
nope, not only these two. there might be fields related to `Kleros Curate Service`, tokenImage and any other .

Reply:
> And not storing them directly in `TokenObject` is just for organization purpose right? Not because there is some kind of sharing (eg. 2 `TokenObject` refers to the same `TokenInfoObject` instance)?

Forgot to send out this one earlier. Sorry.

Reply:
> nope, not only these two. there might be fields related to `Kleros Curate Service`, tokenImage and any other .

And `TokenObject` is always 1:1 with `TokenInfoObject`?

Reply:
> And TokenObject is always 1:1 with TokenInfoObject?

yes, TokenInfoObject creates only one time for each token object